src/get-started/create-service.md

```diff
@@ -670,17 +649,38 @@ while the demo blockchain is a specific example of its usage. For this reason,
 we will position the blockchain code as an [*example*][cargo-example] and
 place it into [`examples/demo.rs`][demo.rs].
 
+### Additional Dependencies
+
+Since services themselves aren't require the Exonum node, in this example we
+want to create one, and interact with it as well. Thus, we have to add several
+additional dependencies in our `Cargo.toml`:
+
+```toml
+# Dependencies required for example.
+exonum-explorer-service = "1.0"
```

Comment:
Off-topic: If `exonum-cli` supported service instantiation, this section could be significantly simplified.

Reply:
I agree, but https://github.com/exonum/exonum-doc/pull/391#discussion_r372386195